src/package-copy.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "",
+    "version": "0.0.0",
+    "description": "",
+    "main": "",
```

```suggestion
    "main": "./prod/index.js",
```

Since this will end up generating a `tsconfig` that compiles `src` into `prod`, and will also provide a blank `index.ts` to compile, we could also add the default entry point.